### PR TITLE
[beta-updates] animations

### PR DESCRIPTION
**Pull request by craiglabenz**
Updates `animations` for Dart 2.17

### CODE REVIEW

File: animations/lib/main.dart [hunk 1]
```diff
@@ -6,6 +6,7 @@ import 'dart:io';
 
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
+// ignore: depend_on_referenced_packages
```
Comment (craiglabenz) [inline animations/lib/main.dart:9]: Ideally, we wouldn't need to continue using this override; but I've never encountered this lint before and am not sure what to do for plugins that come directly from Flutter

Comment (domesticmouse) [reply animations/lib/main.dart:9]: Which package is failing `depend_on_referenced_packages`? By my reading, it's `package:window_size` which definitely should be directly referenced in `pubspec.yaml`, or am I missing something?

Comment (domesticmouse) [reply animations/lib/main.dart:9]: And it is referenced. I'm confused as to why this lint is firing here. @bwilkerson halp?

**Review by domesticmouse — APPROVED** [review]

File: animations/lib/main.dart (hunk 1 above)
Comment (craiglabenz) [reply animations/lib/main.dart:9]: Seems to be a potential bug around git-based dependencies?

**Pull request merged**